tasks/vars_SUSE_SLES12.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+# SLES12 has a different package name
+targetcli_package:
+  - targetcli-fb
+  - python-dbus-python
```

Comment:
addition of `python-dbus-python` is a nice catch. I had my testing already preinstalled with this package, but testing to remove I could see that `targetcli` broke. I would expect this to be handled by packaging system (at least in OSes I use) but here it looks it needs to be specified explicitly. Just wanted to say 'nice job finding this!'.

Reply:
this doesn't need any changes, it is here as 'thanks for noticing this'. Feel free to 'resolve' this after reading.